classifai.php

```diff
@@ -84,6 +84,7 @@ function classifai_define( $name, $value ) {
 function classifai_autoload() {
 	if ( file_exists( CLASSIFAI_PLUGIN_DIR . '/vendor/autoload.php' ) ) {
 		require_once CLASSIFAI_PLUGIN_DIR . '/vendor/autoload.php';
+		require_once CLASSIFAI_PLUGIN_DIR . '/vendor/woocommerce/action-scheduler/action-scheduler.php';
```

Comment:
I wonder if we'd be better off only loading this if we need it. For instance, if a site isn't using a Feature that utilizes this library (in this case, just the Embeddings Feature is using it) seems better to not have the overhead of loading this